lib/redlock/client.rb

```diff
@@ -122,6 +160,12 @@ class RedisInstance
         end
       eos
 
+      PTTL_SCRIPT = <<-eos
+        if redis.call("exists", KEYS[1]) then
```

Comment:
i assume you intentionally put this in, but still have to ask: Is the `exists` before `get` actually needed? Doesn't `get` simply return `nil` if the key isn't present?

Reply:
```
$ redis-cli
127.0.0.1:6379> get 'foo'
(nil)
127.0.0.1:6379> pttl 'foo'
(integer) -2
```

Reply:
You're right, the code works just as well without it. After testing a bit, it probably wasn't even doing what I thought.